### DISCUSSION TITLE
HELP! Simulator paused!

### DISCUSSION THREAD

**Discussion by IHateVolatility** [logisim-evolution/logisim-evolution#991] [Q&A] [answered]
When I was almost finished with my 8 bit cpu, I wanted to test it. But when I tried everything was just not right, the automatic clock didn't work and when I manually pulse the clock the registers don't update. Left up in the screen showed a text saying: Simulator paused: xxx signals changed, xx input changes. I have tried everything what I could think of to continue the simulator without success. Please help me.
I also tried to copied and pasted the whole cpu to a new simulator. But the same thing happened.

![xx](https://user-images.githubusercontent.com/8041294/131697571-070f8058-b3a7-430e-9fb9-5e2ccddb6f6f.png)

**Comment by maehne** [comment]
The information you provide is not enough for diagnosing your issue. If you want to manually step your simulation, you would usually use "Manual Tick Half Cycle" or "Manual Tick Full Cycle" from the "Simulate" menu. Please describe exact steps how to reproduce your problem (+optional a small video illustrating the problem) and provide a .circ file. You'll have to compress it into a ZIP archive to attach it to a message in this forum or an issue.

PS: By the way, you can directly drag-and-drop screenshots into GitHub's markdown editor.

**Comment by IHateVolatility** [comment, marked as answer]
Thanks for the fast reply, I know how to manually step, but the problem is that it does really weird stuff. 
I don't really understand when and what I do when it happens. When I restart the simulation or completely close the program to start it again, it already says: Simulator paused: xxx signals changed, xx input changes.

[sim_core.zip](https://github.com/logisim-evolution/logisim-evolution/files/7091428/sim_core.zip)

**Reply by maehne** [reply]
Your project is incomplete. You need also to provide all .circ Libraries. Otherwise, we cannot open it.

**Reply by IHateVolatility** [reply]
oh yea, I forgot. 

[sim.zip](https://github.com/logisim-evolution/logisim-evolution/files/7091565/sim.zip)

**Reply by IHateVolatility** [reply]
I just solved it. It was just a simple oscillation error. When I changed a shorted connection it I could continue the simulator. Thanks for your time!

**Reply by maehne** [reply]
Glad you were able to resolve the issue yourself!
